api/v1/symphony.go

```diff
@@ -32,8 +32,13 @@ type SymphonySpec struct {
 	// Removing a variation will cause the composition to be deleted!
 	Variations []Variation `json:"variations,omitempty"`
 
-	// Bindings are inherited from all compositions managed by this symphony.
+	// Bindings are inherited by all compositions managed by this symphony.
 	Bindings []Binding `json:"bindings,omitempty"`
+
+	// SynthesisEnv
+	// Copied opaquely into the compositions managed by this symphony.
+	// +kubebuilder:validation:MaxItems:=500
+	SynthesisEnv []EnvVar `json:"synthesisEnv,omitempty"`
```

Comment:
can we use EnvVar from "k8s.io/api/core/v1" directly and no need to defined new EnvVar in env.go?

Reply:
I don't think it's appropriate to use it because that struct supports ValueFrom which allows references to ConfigMaps/Secrets. We're not (and probably won't for a long time) implementing ValueFrom so I don't think we should expose it in the API.